Scripts/AppCenter/build-appcenter-ios-framework.sh

```diff
@@ -41,8 +44,32 @@ cp -f "${SRCROOT}/${FMK_NAME}/Support/iOS.modulemap" "${INSTALL_DIR}/Modules/mod
 # Copies the headers and resources files to the final product folder.
 cp -R "${SRCROOT}/${WRK_DIR}/Release-iphoneos/include/${FMK_NAME}/" "${INSTALL_DIR}/Headers/"
 
-# Uses the Lipo Tool to merge both binary files (i386 + armv6/armv7/armv7s/arm64) into one Universal final product.
-lipo -create "${DEVICE_DIR}/lib${FMK_NAME}.a" "${SIMULATOR_DIR}/lib${FMK_NAME}.a" -output "${INSTALL_DIR}/${FMK_NAME}"
+# Create the arm64e slice in Xcode 10.1 and lipo it with the device binary that was created with oldest supported Xcode version.
+# Move binary that was create with old Xcode to temp location.
+LIB_IPHONEOS_FINAL="${DEVICE_DIR}/lib${FMK_NAME}.a"
+if [ -z "$MS_ARM64E_XCODE_PATH" ] || [ ! -d "$MS_ARM64E_XCODE_PATH" ] ; then
+echo "Environment variable MS_ARM64E_XCODE_PATH not set or not a valid path."
+else
+echo "Building the arm64e slice."
+LIB_IPHONEOS_TEMP_DIR="${DEVICE_DIR}/temp"
+mkdir -p "${LIB_IPHONEOS_TEMP_DIR}"
+mv "${DEVICE_DIR}/lib${FMK_NAME}.a" "${LIB_IPHONEOS_TEMP_DIR}/lib${FMK_NAME}.a"
+
+# Build with the Xcode version that supports arm64e.
+env DEVELOPER_DIR="${MS_ARM64E_XCODE_PATH}" /usr/bin/xcodebuild ARCHS="arm64e" -project "${FMK_NAME}.xcodeproj" -configuration "Release" -target "${TGT_NAME}"
```

Comment:
Are you using `DEVELOPER_DIR` variable? You keep assigning the result to the env variable.

Reply:
I don't understand the question, the way above is how this works. I'm happy to optimize and spend more time on this. Can you please create a bug for this and we can discuss if we want to optimize?

Reply:
Maybe do this with Zander's suggestion?

Reply:
In other words, you are declaring an environment variable here for `DEVELOPER_DIR` but this isn't used anywhere. The line executes a commend and put result (something that is printed on console) to `DEVELOPER_DIR` variable.

Reply:
If you'd like it separately, it is fine.

Reply:
Made a [bug](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/53938)

Reply:
Synced with @clpolet. `xcodebuild` and `lipo` read this variable so we need this one.
Please ignore my initial comment.